shark-cli/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'application'
 
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
+compileKotlin.kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
It's workaround for this https://stackoverflow.com/questions/48988778/cannot-inline-bytecode-built-with-jvm-target-1-8-into-bytecode-that-is-being-bui

Reply:
thx, worth adding that as a comment.